### DISCUSSION TITLE
Welcome to mesa-geo Discussions!

### DISCUSSION THREAD

**Discussion by jackiekazil** [mesa/mesa-geo#44] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by josse7cz** [comment]
Hello, my name is Joseph. I have chosen the topic of my thesis - Evacuation models in Python. I have an idea to create some such model using Mesa Geo. Will it work? Thank you for your answer.

**Reply by Corvince** [reply]
Hi Joseph, welcome to mesa-geo. Can you share some details about your model? Depending on the spatial aspects you might even be fine with the mesa core library for your evacuation model

**Reply by josse7cz** [reply]
Hey. I don't have anything yet. I need an escape model from a flood wave (depends on contours). The model should be able to import a GIS map and then implement agents. Is it possible by mesa geo? Unfortunately I have not been able to run any of your models. Please, can you make it work in replit.com and send me the link? Thx

**Comment by rcriii42** [comment]
Hi, my name is Robert. I am using mesa-Geo to model dredging projects. Really love the library!

**Reply by Corvince** [reply]
Hi Robert, glad you like the library. Do you care about explaining what your model will be about? Maybe we can even extend the library in a way that is useful to you

**Reply by rcriii42** [reply]
Hi Corvince, A year later, I explained my project in this thread: https://github.com/projectmesa/mesa-geo/discussions/139 The recent work has been great!
